### PR TITLE
incall: play greeting sound

### PR DESCRIPTION
**Pull request by fblackburn1**
Depends-on: https://github.com/wazo-platform/wazo-agid/pull/31

Jira-Issue: https://wazo-dev.atlassian.net/browse/WAZO-1738

### CODE REVIEW

File: dialplan/asterisk/extensions_lib_did.conf [hunk 1]
```diff
@@ -20,7 +20,9 @@ same  =   n,AGI(agi://${XIVO_AGID_IP}/check_schedule)
 same  =   n,GotoIf($["${XIVO_SCHEDULE_STATUS}" = "closed"]?CLOSED,1)
 same  =   n,GoSub(xivo-subroutine,s,1(${XIVO_DIDPREPROCESS_SUBROUTINE}))
 same  =   n,Gosub(xivo-global-subroutine,s,1)
-same  =   n,Gosub(forward,s,1(${XIVO_FWD_INCALL_ANSWER_ACTION},${XIVO_FWD_INCALL_ANSWER_ACTIONARG1},${XIVO_FWD_INCALL_ANSWER_ACTIONARG2}))
+same  =   n,GotoIf(${WAZO_GREETING_SOUND}?:nogreeting)
+same  =   n,Playback(${WAZO_GREETING_SOUND})
```
Comment (sboily) [inline dialplan/asterisk/extensions_lib_did.conf:24]: Be careful, playback is not async, so if playback file is 5 minutes, nothing happens when the file is playing.

Comment (fblackburn1) [reply dialplan/asterisk/extensions_lib_did.conf:24]: Yes it was the specification
> As administrator, on incoming call I want to configure a message completely before distributing to destination

**Comment by fblackburn1** [comment]
@pc-m  need to be merged manually

**Comment by fblackburn1** [comment]
regate

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**